R/createFigure4.R

```diff
@@ -222,17 +221,8 @@ secchi.filtered <- secchi.data %>%
 anyDups <- which(duplicated(select(secchi.filtered, Date, dec_lat_va, dec_lon_va, value)))
 
 secchi.filtered <- secchi.filtered[-anyDups,] %>%
-  filter(value > 0) 
+  filter(secchi > 0, secchi < 150) 
 
-unitConversion <- data.frame(units = c("m","ft","cm"),
-                             conversion = c(0.3048,1,30.48),
-                             stringsAsFactors = FALSE)
-
-secchi.filtered <- secchi.filtered %>%
-  left_join(unitConversion, by="units") %>%
-  mutate(convertedValue = value * conversion) %>%
-  filter(convertedValue < 150) 
-  
 group_by(secchi.filtered, area) %>% summarize(nSites = length(unique(wqx.id)))
```

Comment:
are the units converted to meters? If so I gave you the wrong number, @ldecicco-USGS, should be 150 ft or 46m